Breeding-Insight/bi-web
# BI-2373 Add Delete Option to Germplasm Lists View

# Description
**Story:** [BI-2373](https://breedinginsight.atlassian.net/jira/software/c/projects/BI/boards/1?selectedIssue=BI-2373)


# Dependencies
[BI-2372](https://github.com/Breeding-Insight/bi-api/pull/434) [BI-2374](https://github.com/Breeding-Insight/brapi-Java-TestServer/pull/38)

# Testing

- Import a germplasm list to a project
- navigate to the project's germplasm lists table and click on the details link for the imported list
- in the list details table click on manage-list and select delete
- verify that the list no longer appears in the lists table
- navigate to the all-germplasm tab and verify that the individual germplasm contained in the deleted list are still present


# Checklist:

- [] I have create/modified unit tests to cover this change


[BI-2373]: https://breedinginsight.atlassian.net/browse/BI-2373?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[BI-2372]: https://breedinginsight.atlassian.net/browse/BI-2372?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[BI-2374]: https://breedinginsight.atlassian.net/browse/BI-2374?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
